lib/active_graph/node/query/query_proxy_methods_of_mass_updating.rb

```diff
@@ -62,12 +63,13 @@ def add_rels(node_or_nodes, original_ids)
         end
 
         def delete_rels_for_nodes(original_ids, new_ids)
-          ids = original_ids.select { |id| !new_ids.include?(id) }
```

Comment:
using `-` is faster compared to using `select` loop and then checking with `include?` again